wire/message.go

```diff
@@ -58,6 +58,13 @@ const (
 	CmdCFilterV2      = "cfilterv2"
 	CmdGetInitState   = "getinitstate"
 	CmdInitState      = "initstate"
+	CmdMixPR          = "mixpr"
```

Comment:
I'll get these better documented in the code, but for reference here:

PR: pair request
KE: key exchange
CT: ciphertexts
SR: slot reservation dc-net mix
DC: xor dc-net mix (using reserved anonymous slots from the above SR mix)
CM: confirm message (partially-signed mix transaction)
RS: reveal secrets

Reply:
It's also worth noting that most of these come from the original coinshuffle++ paper, but there are some slight changes made.  Not including CT (which is not part of that paper at all), they use:

KE: Key exchange; CM: Commitment; DC: DC-net; RV: Reveal pads; SK: Reveal secret key; CF: Confirmation

The current names in this code do match the current client-server implementation.

Reply:
I think that the wire command strings don't need to be changed, as they are not user visible, or even developer-visible in most situations.

The max character limit is 12, but I would want to stay at 10 or fewer characters, in case a "v2", "v3" is chosen for future protocol versions.

"mixpairreq" is exactly 10 characters, so that could work, but the rest would be difficult to fit with a matching naming pattern.

Reply:
Cool. Thanks for calling out what the abbreviations are for.  It's a good point about trying not to exceed 10 since it allows for vX (although I guess leaving off the v and just appending the version would allow up to 11, but I agree 10 and the vX seems better).

Some suggestions that stay within 10 and I would find to be quite understandable are:

`mixpairreq`
`mixkeyxchg`
`mixcphrtxt`
`mixslotres`
`mixdcnet`
`mixconfmsg`
`mixreveal`


That said, your point about the variable names versus the wire message command strings is also quite reasonable.  I personally think have the wire names more readable aids in specifications of the protocol as well, but I agree that the more important part in terms of reading the code base would be the names of the types themselves.

I know it's more verbose, but I would very much prefer a bit more verbosity at that level.  The variables inside the code itself using the messages could still be pretty heavily abbreviated.  Since everything starts with Mix, any decent editor with code completion really alleviates any concerns about typing.  e.g.  `kxMsg := wire.NewMsgMix<tab>`.